### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by roshanasingh4**
I have completed everything as per the requirement and have followed the figma design to the point.

Hope everything is good.

Thanks for the review.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -314,20 +329,345 @@ p {
...
+  .greeting {
+    display: none;
+  }
+
+  .greeting-desktop {
+    display: block;
+    margin-top: 62.3px;
+    margin-bottom: 0;
+    color: #172b4d;
+  }
```
Comment (AlphaNtihinduka) [inline style.css:491]: - [ ] Kindly please consider adding the size of the heading to `40px`. This is to ensure that you stick to the Figma template design as much as possible (e.g., font, colors, images, text, margins) hence making your portifolio more professinal ☺️.

File: style.css [hunk 2]
```diff
@@ -314,20 +329,345 @@ p {
...
+
+  .headline h1 {
+    font-size: 30px;
+  }
+
+  .headline p {
+    font-size: 14px;
+    line-height: 20px;
+  }
```
Comment (AlphaNtihinduka) [inline style.css:456]: - [ ] Kindly please consider adding the size of the heading paragraph to `20px`. This is to ensure that you stick to the Figma template design as much as possible (e.g., font, colors, images, text, margins) hence making your portifolio more professinal ☺️.
image

File: style.css [hunk 3]
```diff
@@ -314,20 +329,345 @@ p {
...
+
+  .bottom-separator {
+    display: none;
+  }
+
+  .details h2 {
+    margin: 10px;
+    line-height: 25px;
+    font-size: 18px;
+  }
```
Comment (AlphaNtihinduka) [inline style.css:577]: - [ ] Kindly please consider adding the size of the card heading to `24px`. This is to ensure that you stick to the Figma template design as much as possible (e.g., font, colors, images, text, margins) hence making your portifolio more professinal ☺️.
image

File: style.css [hunk 4]
```diff
@@ -314,20 +329,345 @@ p {
...
+
+  .contact_section h2 {
+    font-size: 22px;
+    text-align: inherit;
+    line-height: 32px;
+    width: 90%;
+    font-style: normal;
+    font-weight: 700;
+    margin: 120px 5px 0 80px;
+  }
```
Comment (AlphaNtihinduka) [inline style.css:539]: - [ ] Kindly please consider adding the size of the contact section paragraph to `30px`. This is to ensure that you stick to the Figma template design as much as possible (e.g., font, colors, images, text, margins) hence making your portifolio more professinal ☺️.

Your design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @roshanasingh4 🙋‍♂️
Good job working on implementing the desktop version of your application. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is not yet professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- _Kindly consider the inline review comments._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by binhussen — APPROVED** [review]
Hi @roshanasingh4 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**